scripts/util.py

```diff
@@ -51,8 +64,74 @@ def read_source_root_path():
 def read_proof_root_path():
     return read_path_from_stdin("the 'proofs' directory (usually '.')")
 
-def read_litani_path():
-    return read_path_from_stdin("the litani script")
+def is_litani_submodule_in_path(path):
+    is_in_path = False
+    try:
+        sys.path.insert(1, path)
```

Comment:
I'd really rather not couple this script to how the litani source code is arranged. Can I suggest that you just check that there's a directory called either `litani` or `aws-build-accumulator` at the path? No need to look inside I think.